apps/condo/domains/organization/integrations/sbbol/sync/index.js

```diff
@@ -87,7 +87,6 @@ const sync = async ({ keystone, userInfo, tokenSet, reqId }) => {
         ...dvSenderFields,
         name: userInfo.name || userInfo.OrgName,
         importId: userInfo.userGuid,
-        importRemoteSystem: SBBOL_IMPORT_NAME,
```

Comment:
it looks like impoerId is also redundant

Reply:
UserData object are presenting data from SBBOL side. In sync logic we are using this field to find/create corresponding UserExternalIdentity